website/docs/build-apps/how-to-guides/import-export-app-entities.md

```diff
@@ -0,0 +1,36 @@
+---
+description: A guide that outlines steps on how to export entities from a page to a JSON file and subsequently import it into another page.
+---
+
+# Reuse Application Entities
+
+This guide shows how to export certain entities from a page and import them into another page or application. Reusable entities are widgets, datasources, queries, JS Objects, and custom libraries. 
+
+
+Follow the steps below to export desired application entities and import it into another page or an application.
+
+1. Open the page from which you want to export the entities. Click the vertical dots icon next to the page name for the action menu.
+
+<figure>
+ <img src="/img/reuse-entities-across-pages-action-menu.png" style= {{width:"640px", height:"auto"}} alt="Export entities"/>
+  <figcaption align = "center"><i>Action menu for import and export</i></figcaption>
+</figure> 
+
+2. Select the **Export** option from the action menu. A dialog box will appear, allowing you to choose the entities you want to reuse.
+<figure>
+  <img src="/img/reuse-entities-across-pages-import-export.png" style= {{width:"640px", height:"auto"}} alt="Import or Export entities"/>
```

Comment:
One image will suffice like this
<img width="354" alt="Screenshot 2023-12-06 at 14 08 13" src="https://github.com/appsmithorg/appsmith-docs/assets/32227135/c76e3eef-f96e-4921-82b8-39dbcf530002">